CMakeLists.txt

```diff
@@ -148,7 +150,16 @@ if(UNIX)
 endif(UNIX)
 
 # CMake provided find modules
-find_package(ZLIB)
+if(BUILD_STATIC)
+    message(STATUS "Making a static build")
```

Comment:
These add verbosity to a CMakeLists.txt that otherwise doesn't have any message() directives.  I'd like these and the other three message() directives omitted, please.